src/schemas/json/project.json

```
@@ -265,8 +274,69 @@
               "type": "boolean",
               "description": "Enables/disables concurrent garbage collection.",
               "default": false
+            },
+            "System.GC.HeapCount": {
+              "type": "number",
+              "minimum": 0,
+              "description": "Limits the number of heaps created by the garbage collector."
+            },
+            "System.GC.HeapAffinitizeMask": {
+              "type": "number",
+              "minimum": 0,
+              "description": "Specifies the exact processors that garbage collector threads should use."
+            },
+            "System.GC.GCHeapAffinitizeRanges": {
+              "type": "string",
+              "minLength": 1,
+              "description": "Specifies the list of processors to use for garbage collector threads."
+            },
+            "System.GC.NoAffinitize": {
+              "type": "boolean",
+              "description": "Specifies whether to affinitize garbage collection threads with processors. To affinitize a GC thread means that it can only run on its specific CPU. A heap is created for each GC thread."
+            },
+            "System.GC.HeapHardLimit": {
+              "type": "number",
+              "description": "Specifies the maximum commit size, in bytes, for the GC heap and GC bookkeeping."
+            },
+            "System.GC.HeapHardLimitPercent": {
+              "type": "number",
+              "description": "Specifies the GC heap usage as a percentage of the total memory."
+            },
+            "System.GC.RetainVM": {
+              "type": "boolean",
+              "description": "Configures whether segments that should be deleted are put on a standby list for future use or are released back to the operating system (OS)."
+            },
+            "System.GC.LOHThreshold": {
+              "type": "number",
+              "description": "Specifies the threshold size, in bytes, that causes objects to go on the large object heap (LOH)."
+            },
+            "System.Threading.ThreadPool.MinThreads": {
+              "type": "integer",
+              "minimum": 1,
+              "description": "Sets the minimum number of threads for the thread pool."
+            },
+            "System.Threading.ThreadPool.MaxThreads": {
+              "type": "integer",
+              "minimum": 1,
+              "description": "Sets the maximum number of threads for the thread pool."
             }
           }
+        },
+        "framework": {
+          "description": "Properties of the shared framework to use when activating the application.",
```

```suggestion
          "title": "framework",
          "description": "Properties of the shared framework to use when activating the application.",
```

Almost the same issue as above.
Optional fix: add `title` to enhance VS Code usage experience.